include/boost/json/string.hpp

```diff
@@ -1061,8 +1148,12 @@ class string
 
     //------------------------------------------------------
 
-    /** Return a pointer to the storage associated with the container
+    /** Access the storage associated with the container.
```

Comment:
I always start the brief sentence with "Return" for accessors. If you have a good reason to deviate from that, I'd like to know, and also we would have to make sure they are all changed. I would just leave it at "return" because that's literally what the function does (and everyone understands that).